agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2092,6 +2094,10 @@ bool backhaul_manager::handle_1905_1_message(ieee1905_1::CmduMessageRx &cmdu_rx,
     case ieee1905_1::eMessageType::ASSOCIATED_STA_LINK_METRICS_QUERY_MESSAGE: {
         return handle_associated_sta_link_metrics_query(cmdu_rx, src_mac);
     }
+    case ieee1905_1::eMessageType::AP_METRICS_QUERY_MESSAGE:
+        return handle_ap_metrics_query(cmdu_rx, src_mac);
+    case ieee1905_1::eMessageType::AP_METRICS_RESPONSE_MESSAGE:
+        return handle_ap_metrics_response(cmdu_rx, src_mac);
```

Comment:
I don't think this is needed. This function is only called when the message comes from the bus, not when it comes from a slave. Instead, you have to add the handling to handle_slave_backhaul_message() (which at the moment is also only supporting Intel VS messages).